src/components/widgets/accounts-ratios/accounts-ratios.directive.coffee

```diff
@@ -0,0 +1,114 @@
+#
+# Component generated by Impac! Widget Generator!
+#
+module = angular.module('impac.components.widgets.accounts-ratios', [])
+module.controller('WidgetAccountsRatiosCtrl', ($scope, $q, $filter, ChartFormatterSvc, ImpacTheming) ->
+
+  w = $scope.widget
+
+  # Define settings
+  # --------------------------------------
+  $scope.orgDeferred = $q.defer()
+  $scope.chartDeferred = $q.defer()
+  $scope.timePeriodDeferred = $q.defer()
+  $scope.histModeDeferred = $q.defer()
+  $scope.numeratorOffsetsDeferred = $q.defer()
+  $scope.denominatorOffsetsDeferred = $q.defer()
+
+  settingsPromises = [
+    $scope.orgDeferred.promise,
+    $scope.chartDeferred.promise,
+    $scope.timePeriodDeferred.promise,
+    $scope.histModeDeferred.promise,
+    $scope.numeratorOffsetsDeferred.promise,
+    $scope.denominatorOffsetsDeferred.promise
+  ]
+
+  $scope.totalRatio = 0
+  $scope.totalNumerator = 0
+  $scope.totalDenominator = 0
+  $scope.calculatedNumerator = 0
+  $scope.calculatedDenominator = 0
+  $scope.simulationMode = false
+  $scope.intervalsCount = 0
+  $scope.isPnl = false
+  $scope.forwardParams = { histParams: w.metadata.hist_parameters }
```

Comment:
What is forwardParams ? I can see it contains hist parameters, but I don't get the naming (maybe a comment?)